server/backend/database/memory/database.go

```diff
@@ -1051,6 +1051,9 @@ func (d *DB) FindChangeInfosBetweenServerSeqs(
 	txn := d.db.Txn(false)
 	defer txn.Abort()
 
```

Comment:
How about simply explaining the case when this early return is necessary as a comment?

Reply:
That comment could be useful.
On the other hand, I wonder if developers need to know when the `from > to` condition occurs. Because, since the DB query has the information that `from <= server_seq <= to`, we can intuitively realize that the case `from > to` doesn't exist.
Think it's still better to give more information? @sejongk

Reply:
Actually, retrieving a document can also cause a `from > to` condition.
When a snapshot of a document reflects the most recent state of the document, there are no additional changes to look for.
If we're going to comment out a short description, it would be nice to specify both cases.